rr/Dockerfile

```diff
@@ -0,0 +1,109 @@
+ARG PHP_VERSION=8.0
+ARG RR_VERSION=2.0
+
+# -----------------------------------------------------
+# App Itself
+# -----------------------------------------------------
+FROM php:$PHP_VERSION-fpm-alpine
+
+ARG PORT=9001
+ARG PUBLIC_DIR=public
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  vim \
+  libzip-dev \
+  unzip
```

Comment:
need to add them in `REQUIRED_PACKAGES` since it runs with 
```
# Install Packages
RUN apk add --update --no-cache $REQUIRED_PACKAGES $DEVELOPMENT_PACKAGES
```